app/schemas/selectors/cinematic.js

```diff
@@ -93,6 +106,79 @@ const leftCharacter = shotSetup => (shotSetup || {}).leftThangType
  */
 const rightCharacter = shotSetup => (shotSetup || {}).rightThangType
 
+/**
+ * @param {ShotSetup} shotSetup
+ * @returns {Object|undefined} background Object
+ */
+const backgroundArt = shotSetup => (shotSetup || {}).backgroundArt
+
+/**
+ * @param {Object} o Object that may have slug property
+ * @returns {string|undefined}
+ */
+const slug = o => (o || {}).slug
+
+/**
+ * Returns properties required to place a background Lank.
+ * @param {Object} backgroundArt
+ * @returns {Object|undefined} a background object
+ */
+const background = backgroundArt => {
+  if (!(backgroundArt || {}).type) {
+    return
+  }
+  if (!(backgroundArt.type || {}).slug) {
+    return
+  }
+  const { pos, scaleX, scaleY } = backgroundArt
+
+  return {
+    slug: backgroundArt.type.slug,
+    thang: _.merge(DEFAULT_THANGTYPE(), { pos, scaleX, scaleY })
+  }
+}
+
+/**
+ * @param {Object|undefined} thangType
+ * @returns {Object|undefined} the thang and slug.
+ */
+const getCharacterFromThangTypeSchema = thangType => {
+  if (!(thangType || {}).type) {
+    return
+  }
+  if (!(thangType.type || {}).slug) {
+    return
+  }
+  const slug = thangType.type.slug
+  const { scaleX, scaleY, pos } = thangType
+  return {
+    slug,
+    thang: _.merge(DEFAULT_THANGTYPE(), { scaleX, scaleY, pos })
+  }
+}
+
+/**
+ * Check if this is the hero type.
+ * If so returns the properties associated.
+ * The caller must get the thangType original using:
+ * `me.get('heroConfig').thangType`
+ * @param {Object|undefined} thangType
+ * @returns {Object|undefined} the thang and hero original.
```

Comment:
It looks like we are only returning thang properties like pos, scaleX, scaleY, and not original

Reply:
Good call out. I wonder if JsDoc lets me document the result in the way I saw you documenting the anonymous objects in your excellent PR?

Reply:
Can you tell me more about which code you are referring to from my PR? (I am not able to recall)

Reply:
[This pattern](https://stackoverflow.com/a/6460748) that unfortunately doesn't have a return version!

Reply:
Also fixing the documentation as you suggested :)

Reply:
Oh yeah, it doesnt have a return version. Maybe we can use `typedef` to define properties for return objects if required.